src/main/kotlin/com/exactpro/th2/check1/entities/RuleConfiguration.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
```

Comment:
The current year is not 2020
```suggestion
 * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
```